src/frontend/cypress.config.ts

```diff
@@ -0,0 +1,23 @@
+import { defineConfig } from 'cypress';
+import dotEnv from 'dotenv';
+import dotenvExpand from 'dotenv-expand';
+import { resolve } from 'path';
+
+const myEnv = dotEnv.config({
+  path: resolve(__dirname, '../../.env'),
+});
+dotenvExpand.expand(myEnv);
+
+const { FRONTEND_ADDR = '', NODE_ENV, FRONTEND_PORT = '8080' } = process.env;
+
+export default defineConfig({
+  env: {
+    baseUrl: NODE_ENV === 'production' ? FRONTEND_ADDR : `http://localhost:${FRONTEND_PORT}`,
```

Comment:
Validating the env and the port for local vs CLI execution

Reply:
I think instead of leaving these types of comments on the PR putting them in the source would be more valuable and they would exist in scenarios other than a code review :)

Reply:
I tent do disagree, I feel that the code itself should be self-explanatory but for PRs it is good to present it as a story for the reviewer so even if they don't have enough context the basic flow of changes can be understood